cli/cmd/transformCmd.go

```diff
@@ -95,13 +100,27 @@ func doTransform() error {
 
 	record, err := doOne()
 	if err == io.EOF {
-		fmt.Println("[]")
+		if schema.Header().ParserSettings.NDJSON {
+			fmt.Println("")
+		} else {
+			fmt.Println("[]")
+		}
 		return nil
 	}
 	if err != nil {
 		return err
 	}
-	fmt.Printf("[\n%s", record)
+
+	start := "[\n%s"
+	middle := ",\n%s"
+	end := "\n]"
```

Comment:
Might be able to come up with better names or create a method on parser settings that returns a struct that encapsulates these variables

Reply:
probably `lparen` and `rparen`, and `delim` might be a better naming option?

I'm fine with the current implementation you proposed. But do need to add unit tests - we tried very hard to keep coverage at 100%.

I'm thinking about adding a utility into https://github.com/jf-tech/go-corelib/tree/master/jsons which this omniparser uses extensively, that the utility is a json writer and encapsulates the functionalities you implement here. But that's a later optimization/refactoring. No need for this time. Just unittest coverage.